src/main/java/seedu/mindmymoney/command/ListCommand.java

```diff
@@ -41,15 +58,52 @@ public boolean isExit() {
     /**
      * Prints user's current list of expenditures.
      */
-    @Override
-    public void executeCommand() throws MindMyMoneyException {
+    public void printExpenditureList() throws MindMyMoneyException {
         if (expenditureList.size() == 0) {
             throw new MindMyMoneyException(
-                    "Your list is currently empty! Please add some expenditures to your list first");
+                    "Your expenditure list is currently empty! Please add some expenditures to your list first");
         } else {
             System.out.print(PrintStrings.LINE);
-            System.out.print(listToString());
+            System.out.print(expenditureListToString());
             System.out.println(PrintStrings.LINE);
         }
     }
+
+    public String creditCardListToString() {
+        int indexOfList = 1;
+        String listInString = "";
+        for (CreditCard i : creditCardList.creditCardListArray) {
+            listInString += indexOfList + ". Name:" + i.getNameOfCard() + ". Cashback:" + i.getCashback()
+                    +  "%. Card limit: $" + i.getMonthlyCardLimit() + ". Card balance: $" + i.getBalance() + "\n";
+            indexOfList++;
+        }
+        assert listInString.length() != 0 : "Return string should be non-empty";
+        return listInString;
+    }
+
+    /**
+     * Prints user's current list of credit cards.
+     */
+    public void printCreditCardList() throws MindMyMoneyException {
+        if (creditCardList.size() == 0) {
```

Comment:
Maybe can change to use the .isEmpty() method instead, then can avoid the magic literal 0 also
```suggestion
        if (creditCardList.isEmpty()) {
```